### PR TITLE
Default content settings and module added.

### PR DESCRIPTION
**Pull request by deepaksingh05**

### CODE REVIEW

File: modules/srijan_devportal_content/srijan_devportal_content.info.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Srijan Devportal Content
+description: "Provides demo content for srijan devportal."
+type: module
+core: 8.x
+
```
Comment (vaibhavjain-in) [inline modules/srijan_devportal_content/srijan_devportal_content.info.yml]: Can we remove blank lines ?

File: modules/srijan_devportal_content/srijan_devportal_content.info.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+distribution:
+  name: Srijan Devportal
+
```
Comment (vaibhavjain-in) [inline modules/srijan_devportal_content/srijan_devportal_content.info.yml]: Lets remove this empty line too ?

File: modules/srijan_devportal_content/srijan_devportal_content.install [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?php
```
Comment (vaibhavjain-in) [inline modules/srijan_devportal_content/srijan_devportal_content.install]: I believe, we dont need this file and code. Please revisit.

File: src/Installer/Form/DemoInstallForm.php [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Extension\ModuleInstallerInterface;
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Installer task for Srijan Developer Portal Demo Content.
+ */
+class DemoInstallForm extends FormBase {
```
Comment (vaibhavjain-in) [inline src/Installer/Form/DemoInstallForm.php]: Please change class name to match with profile name.

File: srijan_devportal.info.yml [hunk 5]
```diff
@@ -45,6 +45,8 @@ install:
 - twitter_block
 - views_slideshow
 - views_slideshow_cycle
+- default_content
+- file_entity
```
Comment (vaibhavjain-in) [inline srijan_devportal.info.yml]: I believe, we can move the dependencies in the new module info file.

File: srijan_devportal.install [hunk 6]
```diff
@@ -2,5 +2,17 @@
...
- * Install, update and uninstall functions for the Srijan Devportal install profile.
+ * Install, update/uninstall functions for the Srijan Devportal install profile.
  */
+
+/**
+ * Implements hook_install_tasks().
+ */
+function srijan_devportal_install_tasks(&$install_state) {
+  return [
+    '\Drupal\srijan_devportal\Installer\Form\DemoInstallForm' => [
```
Comment (vaibhavjain-in) [inline srijan_devportal.install]: We would need to change the class name here.

**Review by vaibhavjain-in — CHANGES_REQUESTED** [review]

File: src/Installer/Form/SrijanDemoInstallForm.php [hunk 7]
```diff
@@ -0,0 +1,100 @@
+<?php
```
Comment (vaibhavjain-in) [inline src/Installer/Form/SrijanDemoInstallForm.php]: I believe, it makes more sense to name it that way, like `SrijanDevportalInstallForm`

File: srijan_devportal.install (hunk 6 above)
Comment (vaibhavjain-in) [reply srijan_devportal.install]: This is still pending.

File: modules/srijan_devportal_content/srijan_devportal_content.info.yml (hunk 1 above)
Comment (vaibhavjain-in) [reply modules/srijan_devportal_content/srijan_devportal_content.info.yml]: This is still pending.

**Review by vaibhavjain-in — CHANGES_REQUESTED** [review]

File: srijan_devportal.install (hunk 6 above)
Comment (deepaksingh05) [reply srijan_devportal.install]: done

File: modules/srijan_devportal_content/srijan_devportal_content.info.yml (hunk 1 above)
Comment (deepaksingh05) [reply modules/srijan_devportal_content/srijan_devportal_content.info.yml]: done

File: modules/srijan_devportal_content/srijan_devportal_content.info.yml (hunk 2 above)
Comment (deepaksingh05) [reply modules/srijan_devportal_content/srijan_devportal_content.info.yml]: done

**Pull request merged**